src/main/java/com/puppycrawl/tools/checkstyle/checks/javadoc/JavadocTagContinuationIndentationCheck.java

```diff
@@ -225,6 +225,10 @@ else if (!CommonUtil.isBlank(text.substring(1, offset + 1))) {
         final List<DetailNode> textNodes = new ArrayList<>();
         DetailNode node = JavadocUtil.getFirstChild(descriptionNode);
         while (JavadocUtil.getNextSibling(node) != null) {
+            if (node.getType() == JavadocTokenTypes.HTML_ELEMENT) {
```

Comment:
Cases like "some text <i>word</i> more text" should not trigger he violation.

Reply:
This Check should have nothing to do with HTML tags
https://checkstyle.org/config_javadoc.html#JavadocTagContinuationIndentation
all content of javadoc block tags is simple text that should be intended +4 to visually show javadoc tags

```
/**
 * @tag comment
 *     Indentation spacing is 4.  <p> OK
 *     Indentation spacing is 4. OK </p>
 *     Indentation spacing is 4. OK
 * @param comment
 *     Indentation spacing is 4.  <p> OK
 *     Indentation spacing is 4. OK </p>
 *     Indentation spacing is 4. OK
 */
public class Test {
}
```

Reply:
We are not flagging violation on HTML tags. We are just traversing deep inside tags to get ```LEADING_ASTERISK``` tokens which we call newline nodes. Then we check if that line is indented or not.

> Cases like "some text word more text" should not trigger he violation.

they won't trigger violation if the line is indented correctly. I have added it at the end of our test file.

> @romani this one looks like a false positive: https://checkstyle-diff-reports.s3.us-east-2.amazonaws.com/876ad3a_2021071907/reports/diff/hibernate-orm/index.html#A1
> 
> This is just a text inside `<i>....</i>` it is too much to require indentation in this case.

this is not a false positive as the line isn't indented correctly and violation is justified. Earlier the ```LEADING_ASTERISK```  inside the HTML tag wasn't flagged leading to a false negative.

Reply:
in code change I see now:
```
     * some text <i>word</i> more text. // violation
     *     some text <i>word</i> more text. // ok
```

so case from @pbludov  is covered.